app/models/heads_of_terms_validation_request.rb

```diff
@@ -35,4 +37,8 @@ def audit_api_comment
   def audit_comment
     {created_at:}.to_json
   end
+
+  def term_cancelled_at_now
```

Comment:
Small suggestion: maybe we could call this `cancel_now!` or similar, to make it clear that it's modifying the term?